pkg/audit/manager.go

```diff
@@ -359,7 +373,14 @@ func (am *Manager) auditResources(
 					am.log.Error(err, "Unable to list objects for gvk", "group", gv.Group, "version", gv.Version, "kind", kind)
 					continue kindsLoop
 				}
-
+				// for each batch, make a parent folder
+				subPath = fmt.Sprintf("%d", folderCount)
+				parentDir := path.Join(*outputDir, subPath)
```

Comment:
Do we also need to group by group/version/kind in order to avoid collisions?

E.g. if we have 7 pods and 3 namespaces and a chunk size of 10, we'd have 4 leftover (non-clobbered) pods after writing out the namespaces.

I forget if we are iterating over all possible versions of an object.

Reply:
`folderCount` starts as 0 before we loop thru clusterAPIResources. For each successful `client.List` call, we create a new folderCount which will be used as the folder containing results from that call. 

> if we have 7 pods and 3 namespaces and a chunk size of 10

In this example, we would have 1 folder with 7 pod yamls written to disk assuming namespaces are not excluded.

Reply:
NVM, I see. folderCount is monotonically increasing... it doesn't reset every time we switch to a new kind.